### DISCUSSION TITLE
`404labfr/laravel-impersonate` support?

### DISCUSSION THREAD

**Discussion by jose123v** [owen-it/laravel-auditing#903] [Q&A] [answered]
https://github.com/404labfr/laravel-impersonate

Using [404labfr/laravel-impersonate](https://github.com/404labfr/laravel-impersonate) in [owen-it/laravel-auditing](https://github.com/owen-it/laravel-auditing) saves the impersonated user instead of the impersonating user

Would PR be accepted to add support for [404labfr/laravel-impersonate](https://github.com/404labfr/laravel-impersonate) in user resolver?

**CC:** @MortenDHansen

**Comment by parallels999** [comment, marked as answer]
Related #904
